ui/components/ItemDetail/ItemDetailLoading.tsx

```diff
@@ -0,0 +1,98 @@
+import { LBA_ITEM_TYPE_OLD } from "@/../shared/constants/lbaitem"
+import { Box, Image, Progress, SkeletonCircle, SkeletonText, Text } from "@chakra-ui/react"
+import React, { useEffect, useState } from "react"
+
+const ItemDetailLoading = ({ item }) => {
+  const getNextLoadingIllustration = (currentIllustration) => {
+    const currentIndex = loadingIllustrations.findIndex((ill) => ill.src === currentIllustration.src)
+    return loadingIllustrations[(currentIndex + 1) % loadingIllustrations.length]
+  }
+
+  const loadingIllustrations: { src: string; text: string }[] =
+    item.ideaType === LBA_ITEM_TYPE_OLD.FORMATION
+      ? [
+          {
+            src: "/images/loading/training_description.svg",
+            text: "Chargement du descriptif de la formation",
+          },
+          {
+            src: "/images/loading/training_contact_info.svg",
+            text: "Vérification des coordonnées du centre de formation",
+          },
+          {
+            src: "/images/loading/training_help.svg",
+            text: "Lien vers le simulateur d’aides aux alternants",
+          },
+        ]
+      : [
+          {
+            src: "/images/loading/job_description.svg",
+            text: "Chargement du descriptif de l’offre",
+          },
+          {
+            src: "/images/loading/job_contact_info.svg",
+            text: "Vérification des coordonnées de l’entreprise",
+          },
+          {
+            src: "/images/loading/job_help.svg",
+            text: "Lien vers le simulateur d’aides aux alternants",
+          },
+        ]
+
+  const [currentIllustration, setCurrentIllustration] = useState(loadingIllustrations[0])
+
+  useEffect(() => {
+    let interval: NodeJS.Timeout | null = null
+
+    let iterations = 0
+    let current = currentIllustration
+    interval = setInterval(() => {
+      if (iterations < 5) {
+        current = getNextLoadingIllustration(current)
+        setCurrentIllustration(current)
+      } else {
+        setCurrentIllustration({
+          src: "/images/loading/hourglass.svg",
+          text: "Hum... Ce chargement semble plus long que prévu",
+        })
+        clearInterval(interval)
+      }
+      iterations++
+    }, 1000)
+
+    return () => {
+      if (interval) {
+        clearInterval(interval)
+      }
```

Comment:
```suggestion
        clearInterval(interval)
```
Je pense pas qu'il y ait des cas où `interval` n'est pas défini